spring-cloud-gcp-docs/src/main/asciidoc/spring-integration-storage.adoc

```diff
@@ -20,6 +20,10 @@ Maven coordinates, using Spring Cloud GCP BOM:
     <groupId>org.springframework.cloud</groupId>
     <artifactId>spring-cloud-gcp-pubsub</artifactId>
```

Comment:
Why is Pub/Sub dependency needed? @joaoandremartins

Reply:
Wow, this is a mistake I discovered a long time ago and I think I fixed on a PR that didn't get to be merged.
Needs to be removed asap!